DEVELOPMENT.md

```diff
@@ -345,3 +351,26 @@ To access the Splunk web interface from outside of the cluster, the following st
    export KIND_CLUSTER_NAME=$(kind -q get clusters)
    ```
    **NOTE**: In case you have more than one kind cluster running, manually set the proper one
+
+1. Using the `hack/quick-install.sh` script. If you get this error:
```

Comment:
I think this apply no matter how you install it. I think the more generic version is "operator deployment times does not become ready" or something similar.

Reply:
I've rephrased it so it still points to an specific step of the development setup but it also specifies that the issue is on deploying the operator.